pkgs/by-name/sc/scom/package.nix

```
@@ -0,0 +1,33 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+}:
+stdenv.mkDerivation rec {
+  pname = "scom";
+  version = "1.0";
+
+  src = fetchFromGitHub {
+    owner = "crash-systems";
+    repo = pname;
+    tag = version;
+    hash = "sha256-z/y4SB0R3nxiBGAWLGBsRH0tncDuBxpjy6NGG8ZbIPw=";
+  };
+
+  enableParallelBuilding = true;
+
+  makeFlags = [ "PREFIX=${placeholder "out"}" ];
+
+  postInstall = ''
+    install -Dm755 -t $out/bin ${pname}
```

```suggestion
    install -Dm755 -t $out/bin scom
```
Same with `rev = pname`, here we want the derivation output to not depend on its name (the goal is to have a FOD, aka fixed output derivation)